plugin-code/src/test/java/com/trivago/cluecumber/files/FileIOTest.java

```diff
@@ -37,7 +37,7 @@ public void writeContentToFileStringInvalidTest() throws Exception {
 
     @Test(expected = FileCreationException.class)
     public void writeContentToFileByteArrayInvalidTest() throws Exception {
-        String path = testFolder.getRoot().getPath().substring(1);
+        String path = testFolder.getRoot().getPath().substring(0);
```

Comment:
Why did you change this to `substring(0)`? Because this returns the exact same source string.

Reply:
No it doesn't. The test was failing due to  "InvalidPathException.class" instead of expected "FileCreationException.class".
Expected Path :- C:\Users\rameshwar.shiral\Downloads\cluecumber-report-plugin-master
Actual Path :- :\Users\rameshwar.shiral\Downloads\cluecumber-report-plugin-master

Reply:
I am confused here. I think, substring(index) starts from zero. So "substring(1);" starts from 2nd letter from path.
path with substring(1); returns below path
:\Users\rameshwar.shiral\AppData\Local\Temp\junit1240953755792720228
path with substring(0); returns below path
C:\Users\rameshwar.shiral\AppData\Local\Temp\junit1240953755792720228.
("C" is missing with substring(1);)

Anyway, Currently I have removed the substring(1) and used below code only.

Please correct me if I am missing something in this regard.